content/en/docs/concepts/services-networking/service.md

```diff
@@ -183,6 +183,13 @@ Accessing a Service without a selector works the same as if it had a selector.
 In the example above, traffic is routed to the single endpoint defined in
 the YAML: `192.0.2.42:9376` (TCP).
 
+{{< note >}}
+api-server does not allow proxying to endpoints that are not mapped to pods. Actions such as
```

Comment:
nit:
```suggestion
The Kubernetes API server does not allow proxying to endpoints that are not mapped to pods. Actions such as
```
or, simplifying the logic:
```suggestion
The Kubernetes API server only allow proxyings to endpoints that map to pods. Actions such as
```
We'd write “API server” or kube-apiserver, but not api-server.